wazimap_ng/datasets/models/geography.py

```diff
@@ -30,10 +38,10 @@ class Geography(MP_Node, BaseModel):
     name = models.CharField(max_length=50)
     code = models.CharField(max_length=20)
     level = models.CharField(max_length=20)
-    version = models.CharField(max_length=20, blank=True)
+    versions = models.ManyToManyField(Version, blank=True)
```

Comment:
So Geography 1 ----- * GeographyBoundary
And Geography * ----- * Version
And GeographyBoundary * ------ * Version

What's ensuring that Geographies relate to exactly the versions their boundaries relate to?

Do we even use Geography.versions anywhere?